applications/filestore-backup/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for filestore-backup.
+
+# -- Override the base name for resources
+nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
+fullnameOverride: ""
+
+# -- Resource limits and requests for the filestore-backup pods
+resources: {}
+
+# -- Annotations for the filestore-backup pods
+podAnnotations: {}
+
+# -- Node selector rules for the filestore-backup pods
+nodeSelector: {}
+
+# -- Tolerations for the filestore-backup pods
+tolerations: []
+
+# -- Affinity rules for the filestore-backup pods
+affinity: {}
+
+# The following will be set by parameters injected by Argo CD and should not
+# be set in the individual environment values files.
+image:
+  # -- Filestore-Backup image to use
+  repository: ghcr.io/lsst-sqre/rubin-google-filestore-tool
+  # -- Pull policy for the filestore-backup image
+  pullPolicy: "IfNotPresent"
+  # -- Tag of filestore-backup image to use
+  # @default -- The appVersion of the chart
+  tag: ""
+
+tool:
```

Comment:
I'm not really a fan of `tool` as the top-level key, since it doesn't communicate anything but implies that it does (like there's some part of this chart that deploys something that *isn't* the tool). For cases like this where you're just collecting settings, I would use `config`, which is consistent with lots of other Phalanx application charts.